[2014-07-08 05:09:17] log.ERROR: exception 'ErrorException' with message 'Object of class Illuminate\Pagination\Paginator could not be converted to int' in D:\zend\Apache2\htdocs\Ta_Web\app\storage\views\eecbfdfc468020169e683f00580865e1:16 Stack trace: #0 D:\zend\Apache2\htdocs\Ta_Web\app\storage\views\eecbfdfc468020169e683f00580865e1(16): Illuminate\Exception\Handler->handleError(8, 'Object of class...', 'D:\zend\Apache2...', 16, Array) #1 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(9428): include('D:\zend\Apache2...') #2 D:\zend\Apache2\htdocs\Ta_Web\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\zend\Apache2...', Array) #3 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(9314): Illuminate\View\Engines\CompilerEngine->get('D:\zend\Apache2...', Array) #4 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(9305): Illuminate\View\View->getContents() #5 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(9917): Illuminate\View\View->render() #6 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(9454): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(5122): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 D:\zend\Apache2\htdocs\Ta_Web\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 D:\zend\Apache2\htdocs\Ta_Web\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'view', Object(Illuminate\View\View)) #10 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(4952): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'view', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(8178): call_user_func_array(Object(Closure), Array) #13 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(8165): Illuminate\Routing\Route->callCallable() #14 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(4963): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(534): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 D:\zend\Apache2\htdocs\Ta_Web\bootstrap\compiled.php(521): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 D:\zend\Apache2\htdocs\Ta_Web\public\index.php(49): Illuminate\Foundation\Application->run() #18 D:\zend\Apache2\htdocs\Ta_Web\server.php(19): require_once('D:\zend\Apache2...') #19 {main} [] []
[2014-07-08 05:37:23] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function getClientOriginalName() on a non-object' in D:\zend\Apache2\htdocs\Ta_Web\app\controllers\PenerbitController.php:111 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-08 05:39:49] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function getClientOriginalName() on a non-object' in D:\zend\Apache2\htdocs\Ta_Web\app\controllers\PenerbitController.php:111 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-08 05:45:47] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Penerbit' not found' in D:\zend\Apache2\htdocs\Ta_Web\app\storage\views\6cea172e687435ef61a499f5613826d3:43 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-08 05:47:05] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'App' (T_STRING)' in D:\zend\Apache2\htdocs\Ta_Web\app\storage\views\6cea172e687435ef61a499f5613826d3:1 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-07-08 05:48:25] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'App' (T_STRING)' in D:\zend\Apache2\htdocs\Ta_Web\app\storage\views\6cea172e687435ef61a499f5613826d3:1 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
